app/src/test/java/com/appunite/loudius/di/CheckModulesTest.kt

```diff
@@ -0,0 +1,45 @@
+package com.appunite.loudius.di
```

Comment:
added a CheckModulesTest, which tests the dependencies if all are correctly injected

Reply:
Praise: More useful tests❤️